codegen/index.js

```diff
@@ -20,6 +20,10 @@ const types = {
   "min": "Number",
   "multiple": "Boolean",
   "muted": "Boolean",
+	"onClick": "EventHandler",
+	"onInput": "EventHandler",
+	"onInvalid": "EventHandler",
+	"onSubmit": "EventHandler",
```

Comment:
Not sure if Github is displaying them in a weird way or they are not aligned properly?

Reply:
wtf tabs

Reply:
omg, somehow they slipped in